apistar/main.py

```diff
@@ -27,18 +28,16 @@ def get_current_app():
     if not hasattr(module, 'app'):
         raise ConfigurationError("The app.py module did not contain an 'app' variable.")
 
-    app = module.app
+    app = module.app  # type: ignore
```

Comment:
Not sure how else to do this one since mypy complains about `module` not having an `app` attribute otherwise.